components/data-services/org.wso2.carbon.dataservices.core/src/main/java/org/wso2/carbon/dataservices/core/DBDeployer.java

```diff
@@ -97,14 +97,7 @@
 import java.lang.management.ManagementFactory;
 import java.net.MalformedURLException;
 import java.net.URL;
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Comparator;
-import java.util.Iterator;
-import java.util.Map;
-import java.util.Set;
-import java.util.TreeMap;
-import java.util.TreeSet;
+import java.util.*;
```

Comment:
seems this has fixed with the next commit https://github.com/wso2/carbon-data/pull/148/commits/3fcc75d61e2acaf186859ca904e73dc8bdd99473